Raywire/iReporter
# Ft disable account endpoint 163318333

**What does this PR do?**
Adds functionality for an admin user to disable or enable a user account
**Descriptions of the tasks to be completed?**
Add an endpoint to change a user's active status to True or False
**How should this be manually tested?**
Git clone the repo and cd into the main directory and run the application as per the README file
Using postman on your localhost run PATCH /api/v2/users/`<username>`/activate
The request should contain a body with:
`{
	"isactive": "True"
}`
**Pivotal tracker story**
#163318333
